CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Fix a false positive for `RSpec/Capybara/SpecificMatcher` when may not have a `href` by `have_link`. ([@ydah][])
 * Add `NegatedMatcher` configuration option to `RSpec/ChangeByZero`. ([@ydah][])
 * Add new `RSpec/Capybara/SpecificFinders` cop. ([@ydah][])
+* Add support numblocks for `RSpec/AroundBlock`, `RSpec/EmptyLineAfterHook`, `RSpec/ExpectInHook`, `RSpec/HookArgument`, `RSpec/HooksBeforeExamples` `RSpec/IteratedExpectation` and `RSpec/NoExpectationExample`. ([@ydah][])
```

Comment:
`Add support numblocks for` -> `Add support for numblocks to`

Reply:
`RSpec/HooksBeforeExamples` `RSpec/IteratedExpectation` and `RSpec/NoExpectationExample` ->
`RSpec/HooksBeforeExamples`,`RSpec/IteratedExpectation`, and `RSpec/NoExpectationExample`

Reply:
Thank you very much. I've corrected those.